test/jasmine/tests/click_test.js

```diff
@@ -0,0 +1,55 @@
+var Plotly = require('@src/index');
+var Lib = require('@src/lib');
+
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+var mouseEvent = require('../assets/mouse_event');
+
+
+describe('click event', function() {
+    var mock = require('@mocks/14.json');
+
+    describe('event data', function() {
+        var mockCopy = Lib.extendDeep({}, mock),
+            clientX = 351,
+            clientY = 223,
+            gd;
+
+        function click() {
+            mouseEvent('mousemove', clientX, clientY);
+            mouseEvent('mousedown', clientX, clientY);
+            mouseEvent('mouseup', clientX, clientY);
```

Comment:
@mdtusz @cldougl 

cartesian click events use the hover data from the `mousemove` event and then *simulate* a click event on mouseup [here](https://github.com/plotly/plotly.js/blob/master/src/plots/cartesian/graph_interact.js#L2079-L2081).

So to test `plotly_click` we need to fire three events at the same client (x,y) location.